Sources/SwiftSyntaxBuilder/BuildablesConvenienceInitializers.swift.gyb

```diff
@@ -23,36 +23,36 @@ import SwiftSyntax
 
 % for node in SYNTAX_NODES:
 %   if node.is_buildable():
-%     has_syntax_collection_child = False
+%     should_create_convenience_initializer = False
 %     for child in node.children:
 %       child_node = NODE_MAP.get(child.syntax_kind)
-%       if child_node and child_node.is_syntax_collection():
-%         has_syntax_collection_child = True
+%       child_token = SYNTAX_TOKEN_MAP.get(child.syntax_kind)
+%       if (child_node and child_node.is_syntax_collection()) or (child_token and not child_token.text):
+%         # If node have syntax collection as child, we can create a convenience initializer using a result builder
+%         # If the child token doesn’t have a text, it takes a String as parameter, we can create a convenient initializer
```

Comment:
I think this would be more understandable as 

```suggestion
%         # If the child token doesn’t have a text, we can create a convenience initializer that takes a string.
```